kratos/factories/linear_solver_factory.h

```diff
@@ -151,31 +151,33 @@ inline std::ostream& operator << (std::ostream& rOStream,
 }
 ///@}
 
-typedef TUblasSparseSpace<double> SparseSpaceType;
-typedef TUblasDenseSpace<double> LocalSparseSpaceType;
+typedef TUblasSparseSpace<double> SpaceType;
```

Comment:
Why this renaming?

Reply:
Because in half ok the linear_solver_factory `TUblasSparseSpace` was refered as `SpaceType` and on the other half as `SparceSpaceType`.

Since the same happened to the `LocalSparseSpaceType`, and the name is wrong because should be `LocalDenseSpaceType`, I just removed the Sparse and Dense from the aliases and keep it consistent with the others aliases (for example the ones in the `standard_linear_solvers_factories.h`)

Reply:
Puff, I think we need to clarify notation and then unify.

Reply:
Agree, but clearly defining as `Sparse` something that is `Dense` was wrong